src/ui/server/constants/routes/insurance.ts

```diff
@@ -1,26 +1,27 @@
 import { PRODUCT } from '../product';
 
-const INSURANCE_ROOT = '/insurance';
+export const ROOT = '/insurance';
```

Comment:
Is not having this as INSURANCE_ROOT better? If we export this anywhere (and have root for quote as well)

Reply:
Good shout, updated.